pynicotine/slskproto.py

```diff
@@ -1214,6 +1214,9 @@ def _init_server_conn(self, msg_obj):
         server_socket.setsockopt(socket.SOL_SOCKET, socket.SO_RCVBUF, self.SOCKET_READ_BUFFER_SIZE)
         server_socket.setsockopt(socket.SOL_SOCKET, socket.SO_SNDBUF, self.SOCKET_WRITE_BUFFER_SIZE)
 
+        if hasattr(socket, "TCP_NODELAY"):
```

Comment:
I believe TCP_NODELAY should be available on every OS we support. Do you know one that doesn't have it?

Reply:
Not aware of any without it. I was just copying the safety checks in https://github.com/nicotine-plus/nicotine-plus/blob/5ecf2df5718d5454cb63370556261a343022b561/pynicotine/slskproto.py#L1144-L1177

Reply:
I think we can leave the check out for now.

Reply:
The CPython code itself checks for the corresponding C define: https://github.com/python/cpython/blob/7e894c2f38f64aed9b259c8fd31880f1142a259d/Modules/socketmodule.c#L8561

Reply:
OK. Check removed.